backend/app/controllers/id_lookup_controller.rb

```diff
@@ -2,35 +2,83 @@ class ArchivesSpaceService < Sinatra::Base
 
   Endpoint.get('/repositories/:repo_id/find_by_id/resources')
     .description("Find Resources by their identifiers")
+    .example("shell") do ||
```

Comment:
Is this empty `||` meant to be here? It doesn't seem to do any harm but looked odd to me.

Reply:
Looks like that slipped through. RubyMine has been adding those pipes by default. I can remove it and resubmit the pull request if you would like. I can double check if there's any other places those are showing up.

Reply:
Gotcha - IDEs are almost _too_ helpful these days, if you ask me 😆 Anyway, yes, that would be great if it isn't too much trouble.

Reply:
@donaldjosephsmith Removed those pipes, fixed some indenting issues, and others. Would it be possible to squash commits and resubmit this pull request by any chance?